Makefile

```diff
@@ -109,9 +109,16 @@ fmt:
 	@echo "===> Formatting Go files <==="
 	@gofmt -s -l -w $(GO_FILES)
 
+.PHONY: .linter
+.linter:
+	@if ! command golint; then \
```

Comment:
same as below, I think you should append `$(GOPATH)/bin` to the PATH in case this wasn't done by the user

Reply:
Thanks, added.